.github/ISSUE_TEMPLATE/new-tweet-request.yml

```diff
@@ -0,0 +1,32 @@
+name: New Tweet Request
+description: Create a new tweet request
+title: "[Tweet] <title>"
+body:
+  - type: markdown
+    attributes:
+      value: 
+        Thank You for creating a new tweet request and it will be merged soon and the tweet will go out soon
+  - type: checkboxes
+    attributes:
+      options: 
+      - label: Remember to follow the 280 characters limit to tweet 
+        required: true
+  - type: textarea
+    id: Tweet
+    attributes:
+      label: What do you want to tweet?
+      placeholder: Time to drop a tweet.
+    validations:
+      required: true
```

Comment:
```suggestion
  - type: textarea
    id: Tweet
    attributes:
      label: What do you want to tweet?
      placeholder: Write your tweet here
    validations:
      required: true
  - type: checkboxes
    label: Caution
    attributes:
      options: 
      - label: This tweet follows the 280 characters limit 
        required: true
```

Reply:
`label:` is necessary for `type: checkboxes`, otherwise GitHub will throw an error. Also I have shifted the checkbox after the tweet textbox